vraptor-site/layouts/header.html

```diff
@@ -9,7 +9,7 @@ <h1 class="logo"><a class="logo-link" href="/<%=path%>">
   <div class="header-links">
     <ul class="community">
 
-      <li><a href="https://groups.google.com/forum/<%=path == 'en' ? '?hl=en#!forum/vraptor4' : '#!forum/caelum-vraptor'%>" target="_blank">Google Groups</a></li>
+      <li><a href="https://groups.google.com/forum/#!forum/caelum-vraptor>" target="_blank">Google Groups</a></li>
```

Comment:
Hi @danilomunoz, thanks in advance for sending the pull request.
There is distinct lists for EN and PT_BR languages, as you can see:

https://groups.google.com/forum/#!forum/caelum-vraptor-en
https://groups.google.com/forum/#!forum/caelum-vraptor 

Thinking on that, we can't use `caelum-vraptor` group for both. it'd 
be more appropriate to do something like that:

 ```
/#!forum/caelum-vraptor<%= path == 'en' ? '-en' : ' ' %>
```

What do you think? Could you please update your PR? Thanks again